ubyssey/static/src/js/components/Nationals/Desktop.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import Mvp from './Mvp.jsx'
+import TeamContent from './TeamContent.jsx'
+
+class Desktop extends React.Component {
```

Comment:
Could be made as a functional component, but then again this will not be used again so...